Formula/rdm.rb

```diff
@@ -0,0 +1,97 @@
+class Rdm < Formula
+  desc "Cross-platform GUI management tool for Redis"
+  homepage "https://redisdesktop.com/"
+  url "https://github.com/uglide/RedisDesktopManager/archive/0.9.9.tar.gz"
+  sha256 "8f7244813d5a82bc686a3af9f98e1372479c9861d724ba7e7a2b162ccf35d9bb"
+  head "https://github.com/uglide/RedisDesktopManager.git"
+
+  depends_on "cmake" => :build
+  depends_on :xcode => :build
+  depends_on :macos
+  depends_on :macos => :high_sierra
```

Comment:
Not sure why jenkins is trying to build on sierra when I have the minimum version set to high_sierra here.